db/migrate/20230601065037_create_books.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :books do |t|
+      t.string :name
```

Comment:
You can add validation on name on schema as well, since you desired to have it presented on `Book` model, `validates :name, presence: true`.
```suggestion
      t.string :name, null: false
```